Update User

YahirsGonzalez opened the pull request:
Update UsersController.cs:

user can be updated

YahirsGonzalez committed:
Update User

Update UsersController.cs:

the user can be Updated

gabrielkdc commented on UsersAPI/Controllers/UsersController.cs:
```diff
@@ -38,4 +39,41 @@ public async Task<IActionResult> GetUserDetails(int id)
+        }
+        catch (DbUpdateConcurrencyException)
+        {
+            if (!UserExists(id))
```
Sería mejor validar si el usuario existe antes de intentar actualizarlo.

Así no haces peticiones innecesarias a la BD. Cuando hay una Exception todo el flujo del programa se interrumpe y puedes tener problemas de performance.

YahirsGonzalez committed:
Update User

the exists of the user is validated before try to save changes for a better flow

gabrielkdc committed:
Fix conflicts
